defaults/main.yml

```diff
@@ -112,6 +112,15 @@ telemetry_tls_privatekey:
 telemetry_tls_certificate:
 telemetry_tls_ca_certificate:
 
+# JWT configuration
+jwt_remote_certificate_path: /etc/hub/jwt
+jwt_privatekey:
+jwt_publickey:
+jwt_credentials_path:
+jwt:
+  privatekey: "{{jwt_privatekey}}"
```

Comment:
whitespaces around the interpolated variables (consistency with the rest of the variables in this file)